### PR TITLE
Vardir image

### PR DESCRIPTION
**Pull request by vidarl**
Separating the vardir into it's own image. This is to separate content (db and vardir ) from application

### CODE REVIEW

**Comment by vidarl** [comment]
@andrerom : ready for review....

File: docker-compose_services.yml [hunk 1]
```diff
@@ -2,15 +2,20 @@
 
 
 site1dbvol:
-  image: debian:jessie
+  image: busybox
```
Comment (andrerom) [inline docker-compose_services.yml]: noen spesiell grunn til å begynne å pulle inn busybox?

**Comment by andrerom** [comment]
+1

File: docker-compose_services.yml (hunk 1 above)
Comment (vidarl) [inline docker-compose_services.yml]: Just to decrease the data volumes
Preferrably, it should be "scratch", but docker-compose do not support that ( requires echo command inside container..... )

**Pull request merged**